ydb/docs/en/core/reference/kafka-api/examples.md

```diff
@@ -0,0 +1,127 @@
+# Kafka API usage examples
+
+This article provides examples of usage the Kafka API to work with [topics](../../concepts/topic.md).
+
+Before executing the examples, [create a topic](../ydb-cli/topic-create.md) and [add a consumer](../ydb-cli/topic-consumer-add.md).
+
+## Examples of working with topics
+
+The examples use:
+
+ * `ydb:9093` — host name.
+ * `/Root/Database` — database name.
+ * `/Root/Database/Topic` — topic name.
+ * `user@/Root/Database` — username. Includes the username and database name.
+ * `*****` — user password.
+
+
+## Writing data to a topic
+
+### Writing via Kafka Java SDK
+
+This example includes a code snippet for writing data to a topic via Kafka Java SDK.

```

Comment:
is there a full version of this with imports, main function definition, etc.? a link to something like this would have been useful